quipucords/api/scan/model.py

```diff
@@ -116,7 +116,7 @@ class ScanOptions(models.Model):
     JBOSS_BRMS_EXT = 'jboss_brms_ext'
     JBOSS_WS_EXT = 'jboss_ws_ext'
 
-    max_concurrency = models.PositiveIntegerField(default=50)
+    max_concurrency = models.PositiveIntegerField(default=25)
```

Comment:
Let's just make a constant DEFAULT_MAX_CONCURRENCY=25 at the top and use it everywhere.  I realize that the existing code didn't do that, but this constant will make it easier to search for.